pkg/logs/message/constants.go

```diff
@@ -0,0 +1,59 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2018 Datadog, Inc.
+
+package message
+
+import (
+	"bytes"
+)
+
+// Syslog severity levels,
+// for more information check https://en.wikipedia.org/wiki/Syslog#Severity_level.
+var (
+	SevEmergency = []byte("<40>")
+	SevAlert     = []byte("<41>")
+	SevCritical  = []byte("<42>")
+	SevError     = []byte("<43>")
+	SevWarning   = []byte("<44>")
+	SevNotice    = []byte("<45>")
+	SevInfo      = []byte("<46>")
+	SevDebug     = []byte("<47>")
+)
+
+// Status values
+const (
+	StatusEmergency = "emerg"
+	StatusAlert     = "alert"
+	StatusCritical  = "critical"
+	StatusError     = "error"
+	StatusWarning   = "warning"
+	StatusNotice    = "notice"
+	StatusInfo      = "info"
+	StatusDebug     = "debug"
+)
+
+// SeverityToStatus transforms a severity to a status.
+func SeverityToStatus(severity []byte) string {
```

Comment:
Are you sure there isn't a better way to do this ? I find the switch a bit cumbersome, and maybe not very efficient as it will be evaluated for every message. I think you could try storing the values in a map and doing a simple get, or something more clever.